bi_sql_editor/models/bi_sql_view.py

```diff
@@ -218,10 +226,9 @@ def button_open_view(self):
         return {
             'type': 'ir.actions.act_window',
             'res_model': self.model_id.model,
-            'view_id': self.graph_view_id.id,
             'search_view_id': self.search_view_id.id,
-            'view_type': 'graph',
-            'view_mode': 'graph',
+            'view_type': 'form',
```

Comment:
hi @bealdav, view_type is used until last branches. (V10).
https://github.com/odoo/odoo/blob/10.0/odoo/addons/base/ir/ir_actions.py#L295
but yes, view_type has a default value to 'form'. 

I propose to keep this code, and better to make a batch to remove globally, all that useless keys, in all OCA repository.

regards.